edge-modules/azure-monitor/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG base_tag=3.1.12-bionic-arm32v7
```

Comment:
These two files can be removed:
```
edge-modules/azure-monitor/docker/linux/arm32v7/base/Dockerfile
edge-modules/azure-monitor/docker/linux/arm64v8/Dockerfile
```
The arm32v7 and arm64v8 images are based on `azureiotedge/azureiotedge-module-base:{tag}`, which is built from the Dockerfiles at `edge-modules/SimulatedTemperatureSensor/docker/linux/{arch}/base`.

See [base-image-build-and-publish.yaml#L9](https://github.com/Azure/iotedge/blob/release/1.1/builds/misc/templates/base-image-publish/base-image-build-and-publish.yaml#L9).

Reply:
Removed